### PR TITLE
remove deprecated boot bisect from front-end

### PR DESCRIPTION
**Pull request by alexandrasp**
Boot bisect is not available in back-end api anymore. It is a fix to remove previous uses of
boot bisect in order to remove 400 error messages from kernel ci front-end.

Signed-off-by: Alexandra Pereira <alexandra.pereira@collabora.com>

### CODE REVIEW

File: app/dashboard/utils/route.py [hunk 1]
```diff
@@ -225,7 +225,9 @@ def init():
     )
     add_rule(
         "/boot/id/<regex(\"[A-Za-z0-9]{24}\"):uid>/",
-        view_func=vboot.BootIdView.as_view("boot-id"), methods=["GET"])
```
Comment (gctucker) [inline app/dashboard/utils/route.py]: Was this change really necessary?

**Review by gctucker — COMMENTED** [review]
Overall LGTM, with one minor thing.

We just need a boot regression on staging now to check it's fixing the issue.

File: app/dashboard/utils/route.py (hunk 1 above)
Comment (alexandrasp) [reply app/dashboard/utils/route.py]: Not exactly, but it was not following the code style..

Comment (gctucker) [reply app/dashboard/utils/route.py]: OK I think it's better to not include it in this PR as it's not related to removing the deprecated boot bisection support.

Comment (alexandrasp) [reply app/dashboard/utils/route.py]: updated

**Comment by gctucker** [comment]
Tested OK on staging with previous version of this PR:
https://staging.kernelci.org/boot/id/5e395b96e60551712ac88ee1/

Doing another round of tests on staging before merging.

**Comment by gctucker** [comment]
Tested OK again on staging.
